### ISSUE TITLE
Navbar: button elements opening menus missing haspopup attribute (4.1.2)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#873]
### Expected behavior

Navigation buttons which open menus have aria-haspopup="true" or "menu". 
https://www.w3.org/TR/wai-aria-practices/#menubutton 
### Actual behavior

Buttons have no attribute.
